src/Color.hs

```diff
@@ -5,8 +5,8 @@ import Codec.Picture (PixelRGB8(..))
 
 type Color = (Word8, Word8, Word8)
 
-color2Px :: Color -> PixelRGB8
-color2Px (r, g, b) = PixelRGB8 r g b
+pixelRGB8 :: Color -> PixelRGB8
+pixelRGB8  (r, g, b) = PixelRGB8 r g b
```

Comment:
so this feels like code smell to me. Currently we're using the PixelRGB8, but we could move to another representation at some point, at which point we might want to change this function and it's name. It makes me wonder whether this function should be in the Color module at all. Maybe worth looking at the JuicyPixel lib to see if they have a more generic way of converting the representation of a color to a pixel